### PR TITLE
Replace all references/occurences of the term 'VC HTTP API' with 'VC API'

### PR DESCRIPTION
**Pull request by PatStLouis**
Following the VC-API working group meeting on 2022-08-03, it was clarified to me that there has been a previous voted decision to refer to this spec as the VC-API and not VC-HTTP-API for simplicity. This is a great decision in my opinion however I noticed a lot of artifact remaining through the repository referring to HTTP. To avoid ambiguity I suggest to remove all mention of VC-HTTP-API when referring to this specification. There is still mention of HTTP-API in the diagram directory and in various .png files, otherwise I think I got most of it.

***
<a href="https://pr-preview.s3.amazonaws.com/IDLab-org/vc-api/pull/299.html" title="Last updated on Aug 7, 2022, 10:21 PM UTC (26d63ab)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api/299/7db3c72...IDLab-org:26d63ab.html" title="Last updated on Aug 7, 2022, 10:21 PM UTC (26d63ab)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
     edDraftURI: "https://w3c-ccg.github.io/vc-api/",
 
     // subtitle for the spec
-    subtitle: "A HTTP API for Verifiable Credentials lifecycle management",
+    subtitle: "A API for Verifiable Credentials lifecycle management",
```
Comment (msporny) [inline index.html:48]:
```suggestion
    subtitle: "An HTTP API for Verifiable Credentials lifecycle management",
```

This is the only instance that I think we should keep.

**Review by msporny — CHANGES_REQUESTED** [review]
Overall, +1 to the PR, thank you for raising it!

There is only one minor change request before we can merge, IMHO.

**Review by dlongley — APPROVED** [review]
Thanks for doing this PR! +1 to @msporny's comments.

**Comment by PatStLouis** [comment]
@msporny your suggestion has been committed. Glad to hear this is welcomed. I'm on vacation for 2 weeks starting tomorrow but can have a look at other relevant repositories when I'm back (such as the test-suites and implementations)

**Review by peacekeeper — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**